core/src/test/java/org/apache/iceberg/inmemory/TestInMemoryViewCatalog.java

```diff
@@ -35,6 +35,9 @@ public void before() {
         ImmutableMap.<String, String>builder()
             .put(CatalogProperties.VIEW_DEFAULT_PREFIX + "key1", "catalog-default-key1")
             .put(CatalogProperties.VIEW_DEFAULT_PREFIX + "key2", "catalog-default-key2")
+            .put(CatalogProperties.VIEW_DEFAULT_PREFIX + "key3", "catalog-default-key3")
```

Comment:
that one doesn't seem to be used anywhere

Reply:
I added this line to verify the behavior when we specified a property with both default (L38) and override (L39) properties. I can remove this line if it's unnecessary.

Reply:
nvm, I missed that we have an assertion on this one, so let's keep it